README.md

```diff
@@ -38,6 +38,8 @@ Within the reStructuredText files use the `sphinx_argparse_cli` directive that t
 | prog        | (optional) the module path to where the parser is defined                                                                     |
 | title       | (optional) when provided, overwrites the `<prog> - CLI interface` title added by default and when empty, will not be included |
 | usage_width | (optional) how large should usage examples be - defaults to 100 character                                                     |
+| group_title_prefix | (optional) groups subsections title prefixes - defaults to `<prog>` |
```

Comment:
The way this is formulated may lead me to believe that I can do `<prog> custom` and gets expanded to the prog name and then the word custom. Perhaps it's more appropriate to say, that if not set will use the parsers `prog`. Same for the subcommand below. Alternatively, we can implement the string replace of `<prog>` and `<sub_command>`.

Reply:
> The way this is formulated may lead me to believe that I can do <prog> custom and gets expanded to the prog name and then the word custom. Perhaps it's more appropriate to say, that if not set will use the parsers prog. Same for the subcommand below.

Yep, totally agree.

> Alternatively, we can implement the string replace of <prog> and <sub_command>.

:thinking: That would be useful, especially for subcommands :+1:

Reply:
I've implemented the string replacement for `{prog}` and `{subcommand}`.